style.css

```diff
@@ -495,30 +983,176 @@ body {
     width: 27px;
     height: 40px;
   }
-}
 
-@media (max-width: 1200px) {
-  .first-page {
-    background: linear-gradient(to right, #000 385px, #fff 385px);
-    min-height: 750px;
+  .second-page {
+    min-height: 830px;
   }
 
-  .header {
-    height: 77px;
-    margin-bottom: 64px;
+  .second-page__img {
+    margin-right: 171px;
   }
 
-  .header__logo {
-    padding: 29px 0 0 44px;
+  .second__img_purses {
+    width: 98%;
+    padding: 104px 0 0 125px;
   }
 
-  .header__logo-loco {
-    font-size: 25px;
-    letter-spacing: 2px;
-    margin-right: 200px;
+  .second__wrap {
+    margin: 45px 68px 0 0;
   }
 
-  .header__lang {
+  .second-page__article {
+    width: 340px;
+    padding: 37px 60px 40px 51px;
+    margin-bottom: 34px;
+  }
+
+  .second-page__title {
+    font-size: 38px;
+    letter-spacing: -0.8px;
+  }
+
+  .second-page__subtitle {
+    font-size: 15px;
+    letter-spacing: 0.9px;
+    margin-bottom: 59px;
+  }
+
+  .second-page__description {
+    font: 600 13px/19px 'Montserrat-Regular', sans-serif;
+    margin-bottom: 37px;
+  }
+
+
+  .second-page__color-options {
+    margin-bottom: 43px;
+  }
+
+  .eclipse {
+    width: 28px;
+    height: 28px;
+  }
+
+  .second-page__price {
+    margin-right: 10px;
+    margin-bottom: 49px;
+    font-size: 37px;
+  }
+
+  .second__quantity__input {
+    width: 18px;
+  }
+
+  .second__add-to-cart {
+    font-size: 12px;
+    padding: 3px 39px;
+    width: 233px;
+  }
+
+  .second__more-items {
+    font-size: 12px;
+  }
+
+  .second__more-vector {
+    padding-left: 8px;
+    width: 50%;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 358px, #000 358px, #000 425px, #EBE3D8 425px);
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)